src/qt/rpcconsole.cpp

```diff
@@ -750,6 +754,9 @@ void RPCConsole::setClientModel(ClientModel *model, int bestblock_height, int64_
         ui->startupTime->setText(model->formatClientStartupTime());
         ui->networkName->setText(QString::fromStdString(Params().NetworkIDString()));
 
+        //Setup Peer and Ban Table Splitter
+        ui->tableSplitter->setChildrenCollapsible(false);
+
```

Comment:
This code could be a part of the `debugwindow.ui` file as it is for the horizontal `splitter`.

Qt Designer tool could very helpful for such tasks.

Reply:
Addressed in 0c364201a2e16355cd6424c6137761c7967db4e0